### PR TITLE
Admin should be able to disable an account for various reasons. #184651337

### PR DESCRIPTION
**Pull request by kagame8732**
#### **What does this PR do?**
Admin should be able to disable an account for various reasons.

#### **Description of Task to be completed?**
Due to varying reasons e.g fraud, abuse, 
malicious use, or violation of terms of service, 
the admin should be able to disable any user account.

#### **How should this be manually tested?**
-Open your favorite code editor 
- Navigate to the project and clone it 
- In the terminal type: 
- `npm install` to install the required dependencies.
- `npm start` to start a server
- Account by default, the status will be true 
- Choose an account to be disabled by using userID
- Add userID after this URL `http://localhost:3000/api/disableAccount/`
- Update the account status from isEnabled: **`true`** to isEnabled: **`false`**
- After the above process the account will be enabled/disable

#### What are the relevant pivotal tracker/Trello stories?
[#184651337](https://www.pivotaltracker.com/n/projects/2633635)

### CODE REVIEW

**Review by heshima18 — APPROVED** [review]

**Review by bahati10 — DISMISSED** [review]

File: src/Database/migrations/20230329212027-create-products-table.js [hunk 1]
```diff
@@ -2,7 +2,11 @@
 /** @type {import('sequelize-cli').Migration} */
 module.exports = {
   async up(queryInterface, Sequelize) {
+<<<<<<< HEAD:src/Database/migrations/20230329212027-create-products-table.js
     await queryInterface.createTable('products', {
+=======
```
Comment (Raphaela664) [inline src/Database/migrations/20230329212027-create-products-table.js]: There seems to be some errors due to rebasing

**Review by bahati10 — DISMISSED** [review]

**Review by ruberwa — DISMISSED** [review]
- It would be better if you added authentification, so admin can be authenticated before performing any action.

**Review by ruberwa — DISMISSED** [review]
- It would be better if you added authentification, so admin can be authenticated before performing any action.

**Review by bahati10 — DISMISSED** [review]

**Review by nshutip — COMMENTED** [review]
The task is well implemented but you need to resolve the conflict in branches before merging.

**Review by JulesNIYOMWUNGERI — APPROVED** [review]
well done!

**Review by Raphaela664 — APPROVED** [review]
Great work on this PR!

**Pull request merged**